src/main/java/taxi/model/Driver.java

```diff
@@ -39,6 +47,22 @@ public void setLicenseNumber(String licenseNumber) {
         this.licenseNumber = licenseNumber;
     }
 
+    public String getLogin() {
+        return login;
+    }
+
+    public void setLogin(String login) {
+        this.login = login;
+    }
+
+    public String getPassword() {
+        return password;
+    }
+
+    public void setPassword(String password) {
+        this.password = password;
+    }
+
     @Override
     public boolean equals(Object o) {
```

Comment:
maybe we better include new fields login and password to the equals() and hashcode() methods?